packages/item/src/vaadin-item-styles.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/component-base/src/style-props.js';
+import { css } from 'lit';
+
+export const itemStyles = css`
+  @layer base {
+    :host {
+      align-items: center;
+      border-radius: var(--vaadin-item-border-radius, var(--_vaadin-radius-m));
+      box-sizing: border-box;
+      cursor: pointer;
+      display: flex;
+      gap: var(--vaadin-item-gap, 0 var(--_vaadin-gap-container-inline));
+      height: var(--vaadin-item-height, auto);
+      padding: var(--vaadin-item-padding, var(--_vaadin-padding-container));
+    }
+
+    :host([focus-ring]) {
```

Comment:
This won't apply to combo-box items since we use `focused` attribute there, not `focus-ring`.
Maybe this could be aligned to make reusing styles easier. @jouni WDYT?